File: 2ndPeriod/aedsII/clike/inout/io.cpp
#include <iostream>

using namespace std;

int main(void)
{
    char name[20];
    int age;
    float height;

    cout << "Enter your name: ";
    cin >> name;
    cout << "Enter your age: ";
    cin >> age;
    cout << "Enter your height: ";
    cin >> height;

    cout << name << " is "<< age <<" years old and is "<< height <<"m tall";
    return 0;
}

File: 2ndPeriod/aedsII/clike/inout/io.cs
using System;

class MainClass
{
    public static void Main(string[] args)
    {
        //c# have native string
        string name;
        int age;
        float height;

        Console.Write("Enter your name: ");
        name = Console.ReadLine(); //default string
        Console.Write("Enter your age: ");
        age = int.Parse(Console.ReadLine()); //a conversion is necessary
        Console.Write("Enter your height: ");
        height = float.Parse(Console.ReadLine());

        Console.WriteLine("\n"+name+" is "+age+" years old and is "+height+"m tall");

    }
}